packages/cli/src/commands/serve.ts

```diff
@@ -0,0 +1,130 @@
+import { Command, flags } from '@oclif/command'
+import { ChildProcess, fork } from 'child_process'
+import chalk from 'chalk'
+import chokidar from 'chokidar'
+import ora from 'ora'
+import path from 'path'
+import { prompt } from '../lib/prompt'
+import { sortBy, mergeWith } from 'lodash'
+import { manifest as cloudManifest, ManifestEntry as CloudManifest } from '@segment/action-destinations'
+import { manifest as browserManifest, ManifestEntry as BrowserManifest } from '@segment/browser-destinations'
+
+const manifest: Record<string, CloudManifest | BrowserManifest> = mergeWith(
+  {},
+  cloudManifest,
+  browserManifest,
+  (objValue: any, srcValue: any) => {
+    if (Object.keys(objValue?.definition?.actions ?? {}).length === 0) {
+      return
+    }
+
+    for (const [actionKey, action] of Object.entries(srcValue.definition?.actions ?? {})) {
+      if (actionKey in objValue.definition.actions) {
+        throw new Error(
+          `Could not merge browser + cloud actions because there is already an action with the same key "${actionKey}"`
+        )
+      }
+
+      objValue.definition.actions[actionKey] = action
+    }
+
+    return objValue
+  }
+)
+
+export default class Serve extends Command {
+  private spinner: ora.Ora = ora()
+
+  static description = `Starts a local development server to test your integration.`
+
+  static examples = [`$ ./bin/run serve`, `$ PORT=3001 ./bin/run serve`]
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    directory: flags.string({
+      char: 'd',
+      description: 'destination actions directory',
+      default: './packages/destination-actions/src/destinations'
+    })
+  }
+
+  async run() {
+    const { argv, flags } = this.parse(Serve)
+
+    const { metadataId } = await prompt<{ metadataId: string }>({
+      type: 'select',
+      name: 'metadataId',
+      message: 'Choose a definition:',
+      choices: sortBy(Object.entries(manifest), '[1].definition.name').map(([metadataId, entry]) => ({
+        title: entry.definition.name,
+        value: metadataId
+      }))
+    })
+
+    const destination = manifest[metadataId].directory
+    const folderPath = path.join(process.cwd(), flags.directory, destination)
+
+    let child: ChildProcess | undefined
```

Comment:
```suggestion
    let child: ChildProcess | null = null
```
nit: Since we're assigning a value on L88, `null` is recommended as opposed to `undefined.